public/js/selfoss-events-entriestoolbar.js

```diff
@@ -49,10 +49,10 @@ selfoss.events.entriesToolbar = function(parent) {
         if (parent.find('.entry-toolbar').has('button.entry-share' + shares[0]).length == 0) {
             // add the share toolbar entries
             parent.find('.entry-smartphone-share button.entry-newwindow').after(selfoss.shares.buildLinks(shares, function(name) {
-                return '<button class="entry-share entry-share' + name + '" title="' + name + '"><img class="entry-share" title="' + name + '" src="images/' + name + '.png" height="16" width="16">' + name + '</button>';
```

Comment:
> Why use `aria-haspopup`? It opens a new tab/window, not a popup.

You are right, but, since it behaves like link, either it should be changed to link, or if it is required to keep it button, then aria-haspopup is best bet for this. Since, normal buttons are not supposed to open any new tab/page

Reply:
I will defer to you on this. Unfortunately, we cannot use link, as some sharers work in the background (e.g. copy to clipboard, share with operating system menu).

Maybe we could improve the button labels.

Reply:
> I will defer to you on this. Unfortunately, we cannot use link, as some sharers work in the background (e.g. copy to clipboard, share with operating system menu).
> 
> Maybe we could improve the button labels.

No problem, we can improve the labels.
Should I do something or you're already doing?

Reply:
I will push the changes shortly.